File: src/Entity/Commande.php
<?php

namespace App\Entity;

use App\Repository\CommandeRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity(repositoryClass=CommandeRepository::class)
 */
class Commande
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\OneToOne(targetEntity=Produit::class)
     * @ORM\JoinColumn(nullable=false)
     */
    private $produit;

    /**
     * @ORM\Column(type="integer")
     */
    private $prix;

    /**
     * @ORM\Column(type="integer")
     */
    private $IdUser;

    /**
     * @ORM\Column(type="integer")
     */
    private $totale;

    /**
     * @ORM\Column(type="integer")
     */
    private $numero;

    /**
     * @ORM\Column(type="date")
     */
    private $date;

    /**
     * @ORM\OneToMany(targetEntity=LigneCommande::class, mappedBy="Commande")
     */
    private $ligneCommandes;

    public function __construct()
    {
        $this->ligneCommandes = new ArrayCollection();
    }





    public function getId(): ?int
    {
        return $this->id;
    }

   /* public function getCommande(): ?Produit
    {
        return $this->Commande;
    }

    public function setCommande(?Produit $Commande): self
    {
        $this->Commande = $Commande;

        return $this;
    }*/

   public function getProduit(): ?Produit
   {
       return $this->produit;
   }

   public function setProduit(?Produit $produit): self
   {
       $this->produit = $produit;

       return $this;
   }

   public function getPrix(): ?int
   {
       return $this->prix;
   }

   public function setPrix(int $prix): self
   {
       $this->prix = $prix;

       return $this;
   }

    public function getNumero(): ?int
    {
        return $this->Numero;
    }

    public function setNumero(int $Numero): self
    {
        $this->Numero = $Numero;

        return $this;
    }


    public function getTotale(): ?int
    {
        return $this->totale;
    }

    public function setTotale(int $totale): self
    {
        $this->totale = $totale;

        return $this;
    }

    public function getIdUser(): ?int
    {
        return $this->IdUser;
    }

    public function setIdUser(int $IdUser): self
    {
        $this->IdUser = $IdUser;

        return $this;
    }

   public function getDate(): ?\DateTimeInterface
   {
       return $this->date;
   }

   public function setDate(\DateTimeInterface $date): self
   {
       $this->date = $date;

       return $this;
   }

   /**
    * @return Collection|LigneCommande[]
    */
   public function getLigneCommandes(): Collection
   {
       return $this->ligneCommandes;
   }

   public function addLigneCommande(LigneCommande $ligneCommande): self
   {
       if (!$this->ligneCommandes->contains($ligneCommande)) {
           $this->ligneCommandes[] = $ligneCommande;
           $ligneCommande->setCommande($this);
       }

       return $this;
   }

   public function removeLigneCommande(LigneCommande $ligneCommande): self
   {
       if ($this->ligneCommandes->removeElement($ligneCommande)) {
           // set the owning side to null (unless already changed)
           if ($ligneCommande->getCommande() === $this) {
               $ligneCommande->setCommande(null);
           }
       }

       return $this;
   }





}

File: src/Entities/Commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entities;

import java.util.Date;

/**
 *
 * @author Administrateur
 */
public class Commande {
    
   private int id;
   
   private int produit_id ;
    
 private int prix;
  private int id_user;
  private int totale; 
  private int numero ; 
  private Date date;

    public Commande(int id, int produit_id, int prix, int id_user, int totale, int numero, Date date) {
        this.id = id;
        this.produit_id = produit_id;
        this.prix = prix;
        this.id_user = id_user;
        this.totale = totale;
        this.numero = numero;
        this.date = date;
    }

    public Commande(int produit_id, int prix, int id_user, int totale, int numero, Date date) {
        this.produit_id = produit_id;
        this.prix = prix;
        this.id_user = id_user;
        this.totale = totale;
        this.numero = numero;
        this.date = date;
    }

    public Commande() {
    }

   

    public Commande(int id) {
        this.id = id;
    }

   

    @Override
    public String toString() {
        return "Commande{" + "id=" + id + ", produit_id=" + produit_id + ", prix=" + prix + ", id_user=" + id_user + ", totale=" + totale + ", numero=" + numero + ", date=" + date + '}';
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public int getProduit_id() {
        return produit_id;
    }

    public void setProduit_id(int produit_id) {
        this.produit_id = produit_id;
    }

    public int getPrix() {
        return prix;
    }

    public void setPrix(int prix) {
        this.prix = prix;
    }

    public int getId_user() {
        return id_user;
    }

    public void setId_user(int id_user) {
        this.id_user = id_user;
    }

    public int getTotale() {
        return totale;
    }

    public void setTotale(int totale) {
        this.totale = totale;
    }

    public int getNumero() {
        return numero;
    }

    public void setNumero(int numero) {
        this.numero = numero;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }
       
  
  
  
  
  
}